### PR TITLE
Added the view Calendar

### LINKED ISSUE #32: Como Usuario, Quisiera Acceder a Un Calendario En La App Para Visualizar Los Eventos Disponibles  /  As A User, I Would Like To Access A Calendar In The App To View The Available Events

### PR DESCRIPTION
**Pull request by heliomar-pena**
## Explanation

In this view the user can see the events where he's signed, filtering it by the day.

## Summary

- Added Calendar view and styles similar to the design shared by Daniel in Rock
- Added the queries to get the events of a user, based on [this documentation](https://hasura.io/blog/working-with-dates-time-timezones-graphql-postgresql/) 
- Added storybook and examples with mock data
- Added functionality example in Storybook using the mock data

## Screenshots

In deployment:
imagen

In storybook:
lastrecord

## How to test

Deployment:
1. Go to deployment link
2. Click in the button "Calendario"

Storybook:
1. Go to storybook deployment link
2. Follow the next steps:
imagen
3. Test in other mobile's screen size

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/EventsCalendar/index.vue [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+
+  events.value = mock_events.filter((event) => {
+    const { date: eventDateString } = getDateFromTimestamptz(event.date);
+    const eventDate = new Date(eventDateString);
+
+    return eventDate.getTime() === selectedDate.getTime();
+  });
+};
+
+// Reeplace mock data for database data
```
Comment (Israel-Laguan) [inline src/pages/EventsCalendar/index.vue]: 👀

File: src/pages/EventsCalendar/index.vue [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
 
 const events = ref<Event[]>([]);
-const eventsDates = ref<string[]>([]);
+const eventsDates = ref<string[]>(events_dates);
 
-// Add here the logic to get the events for the current user and save it in events and eventsDates variables
+const selectDate = (selectedDateString: string) => {
+  const selectedDate = new Date(selectedDateString);
+
+  events.value = mock_events.filter((event) => {
```
Comment (Israel-Laguan) [inline src/pages/EventsCalendar/index.vue]: please include the mock into a function that later can be substituted to a call to hasura

Comment (Israel-Laguan) [reply src/pages/EventsCalendar/index.vue]: like

getEvents = () => {
events.value=mock
}

Comment (heliomar-pena) [reply src/pages/EventsCalendar/index.vue]: thanks for the suggestion!, I made the change, please check

**Review by Israel-Laguan — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

**Comment by angelik0828** [comment]
## Validation is done

- The calendar is observed.
- You can see the details of each date in blue
- When there is no event the calendar is one color
- Clicking on the blue date below shows the details of the event

**- Design guide image**
imagen

**Current result**
imagen
imagen

## ⚠️ I think the only thing is that clicking on the word start does not work only if you click on the icon or back arrow.

**Review by angelik0828 — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
@angelik0828 @victor-heliomar maybe we can check this in another issue

image

**Pull request merged**